### DISCUSSION TITLE
Loading tile map and elevation at the same time will slow the loading of tiles

### DISCUSSION THREAD

**Discussion by qiaonifengxue** [SimBlocks/OneWorldSDKforUnity#16] [General]
There is already data in my image tile database. When online elevation data is loaded at the same time, the tile data will be loaded slowly. Tiles should be read directly from the database and will not occupy bandwidth with elevation。How to solve this problem

**Comment by berkcanekici** [comment]
Hello @qiaonifengxue, I am also trying to upload my own imaginary database file, but I could not succeed. Could you explain how you did it? I am constantly getting the error that the tile could not be loaded.
